configure-storybook.md

```diff
@@ -0,0 +1,96 @@
+# Configure your project
+
+Because Storybook works separately from your app, you'll need to configure it for your specific stack and setup. Below, explore guides forconfiguring Storybook with popular frameworks and tools. If you get stuck, learn how you can ask for help from our community.
+
+![A wall of logos representing different styling technologies](./assets/styling.png)
```

Comment:
looks like their images were above the heading they're associated with. can we move these below the headings?